docs/UserGuide.md

```diff
@@ -165,7 +165,7 @@ Format: `list`
 
 [Return to Top](#table-of-contents-toc)
 
-### 3.3 [Coming soon] Listing all [tags](#53-tag): `list t/`
+### 3.3 Listing all tags [tags](#53-tag): `list t/`
```

Comment:
I think tags is being repeated twice here! Should we keep `[tags](#53-tag)` and remove `tags`?

Reply:
Good catch! Will change!